cross-lib/update_books.sh

```diff
@@ -0,0 +1,24 @@
+#!/bin/bash
+
+# Files
+out_file='script/cross-lib/book_data/AUTO_books.txt'
+ubl='script/cross-lib/book_data/USER_ubl.txt'
+mkdir -p 'script/cross-lib/book_data'; touch $out_file; touch $ubl
+
+# Refresh ABL
+abl="$(curl -Ss 'https://raw.githubusercontent.com/openstax/content-manager-approved-books/main/approved-book-list.json')"
+query=$(cat << 'EOF'
```

Comment:
Both ways are **equally correct**, but I prefer simple single quotes to 'here documents' in this case.
```bash
# shellcheck disable=SC2016
query='
  .approved_books
  | .[]
  | .repository_name as $repo
  | .versions[0].commit_metadata.books
  | .[]
  | $repo + " " + .slug
'
```

Reply:
(note: this file has been deleted)